packages/strapi-admin/admin/src/containers/NewNotification/reducer.js

```diff
@@ -0,0 +1,46 @@
+import produce from 'immer';
+import { get } from 'lodash';
+import { SHOW_NEW_NOTIFICATION, HIDE_NEW_NOTIFICATION } from './constants';
+
+const initialState = {
+  notifications: [],
+};
+
+const notificationReducer = (state = initialState, action) =>
+  // eslint-disable-next-line consistent-return
+  produce(state, draftState => {
+    switch (action.type) {
+      case SHOW_NEW_NOTIFICATION: {
+        draftState.notifications.push({
+          // No action.config spread to limit the notification API and avoid customization
+          id: action.id,
+          type: get(action, ['config', 'type'], 'success'),
+          message: get(action, ['config', 'message'], {
+            id: 'notification.success.saved',
+            defaultMessage: 'Saved',
+          }),
+          title: get(action, ['config', 'title'], null),
+          link: get(action, ['config', 'link'], null),
+          timeout: get(action, ['config', 'timeout'], 2500),
+          blockTransition: get(action, ['config', 'blockTransition'], false),
+          uid: get(action, ['config', 'uid'], null),
+          onClose: get(action, ['config', 'onClose'], null),
+        });
+        break;
```

Comment:
You should increment the `notifId` here.

`draftState.notifId = state.notifId + 1`